scopes/harmony/graphql/graphql.main.runtime.ts

```diff
@@ -206,7 +206,7 @@ export class GraphqlMain {
         imports: moduleDeps,
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part